src/utils/getContract.ts

```diff
@@ -0,0 +1,25 @@
+import { AddressZero } from '@ethersproject/constants'
```

Comment:
nit: I would rename this file `contract.ts` as it has more helper methods than `getContract`

Reply:
It would also be very beneficial to add some tests while we're here

Reply:
testing this is pretty intense, feels like a bit of creep but i can add to backlog

Reply:
(given the scope of this ticket is unrelated, smaller in scope, and those contracts seem to have a lot of parts involved with them)